e2e-tests/create-cluster_test.go

```diff
@@ -81,6 +81,22 @@ var _ = Describe("CreateCluster", func() {
 
 		}()
 
+		cmd := exec.Command(tests.KubectlPath, "get", "machine", "-n", namespace)
+		stdout, _ := tests.RunCmd(cmd)
+		print(stdout)
+
+		cmd = exec.Command(tests.KubectlPath, "get", "vm", "-n", namespace)
+		stdout, _ = tests.RunCmd(cmd)
+		print(stdout)
+
+		cmd = exec.Command(tests.KubectlPath, "get", "vmi", "-n", namespace)
+		stdout, _ = tests.RunCmd(cmd)
+		print(stdout)
+
+		cmd = exec.Command(tests.KubectlPath, "get", "datavolume", "-n", namespace)
+		stdout, _ = tests.RunCmd(cmd)
+		print(stdout)
+
```

Comment:
What does this code test?
This is even not failing the test in case of error

Reply:
This is was used for my debug, I removed it completely now.